demo/client/wh_demo_client_crypto.c

```diff
@@ -193,7 +193,7 @@ int wh_DemoClient_CryptoCurve25519(whClientContext* clientContext)
     curve25519_key curve25519PublicKey[1];
     WC_RNG rng[1];
 
-    /* initialize rng to make the cruve25516 keys */
+    /* initialize rng to make the cruve25519 keys */
```

Comment:
@jpbland1 Somehow I didn't notice the double typo the first time XD
```suggestion
    /* initialize rng to make the curve25519 keys */
```